usdigitalresponse/usdr-gost
# fix: standardize client-side vscode debugging (783)

There are no current .vscode files checked into the repo to aid in debugging client-side code. It would be helpful to have these in order to better standardize debugging procedures, at least for those running VSCode.

This is a simple first pass. To use it:
1. Open usdr-gost.code-workspace
2. Go to the "Run and Debug" menu
3. Select "Debug in Chrome (Client)". Under the hood this will run a dev server, and open a debug copy of Chrome.
4. Set breakpoints in client-side code, see that they're hit!

References:
1. [Debugging in VSCode](https://v2.vuejs.org/v2/cookbook/debugging-in-vscode.html)
2. [microsoft/vscode-recipes]( https://github.com/microsoft/vscode-recipes/tree/master/vuejs-cli)

### Ticket #
#783 

### Description
See above. 

### Screenshots / Demo Video
<img width="1661" alt="Screenshot 2023-01-10 at 1 55 27 AM" src="https://user-images.githubusercontent.com/120750336/211482292-59af1aff-56bd-4878-b923-90607da1c490.png">

### Testing
1. Open usdr-gost.code-workspace
2. Go to the "Run and Debug" menu
3. Select "Debug in Chrome (Client)". Under the hood this will run a dev server, and open a debug copy of Chrome.
4. Set breakpoints in client-side code, see that they're hit!

### Checklist
